app/controllers/EoriDetailsController.scala

```diff
@@ -16,25 +16,46 @@
 
 package controllers
 
-import javax.inject.{Inject, Singleton}
 import play.api.libs.json.Json
 import play.api.mvc.{Action, AnyContent, ControllerComponents}
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 
+import javax.inject.{Inject, Singleton}
 import scala.concurrent.Future
 
 @Singleton()
-class ImporterAddressController @Inject()(cc: ControllerComponents)
+class EoriDetailsController @Inject()(cc: ControllerComponents)
   extends BackendController(cc) {
 
   def onLoad(id: String): Action[AnyContent] = Action.async { implicit request =>
```

Comment:
Action definition needs expanding to capture additional parameters expected by *real* endpoint.
```suggestion
  def onLoad(regime: String, acknowledgementReference: String, EORI: String): Action[AnyContent] = Action.async { implicit request =>
```